manifest.php

```diff
@@ -25,12 +25,12 @@
     'label' => 'extension-tao-testqti-previewer',
     'description' => 'extension that provides QTI test previewer',
     'license'     => 'GPL-2.0',
-    'version' => '2.9.0',
+    'version' => '3.0.0',
     'author' => 'Open Assessment Technologies SA',
     'requires' => [
         'tao'          => '>=37.6.0',
         'taoTests'     => '>=12.0.0',
-        'taoItems'     => '>=6.0.0',
+        'taoItems'     => '>=10.2.0',
```

Comment:
Could u merge develop in children PR, and update dependencies here pls

Reply:
checked and fixed